eslzArm/managementGroupTemplates/policyDefinitions/china/mcPolicies.json

```diff
@@ -6985,7 +7015,7 @@
             "mode": "All",
             "description": "This policy denies the creation of a subnet with out a User Defined Route.",
```

Comment:
"This policy denies the creation of a subnet without a User Defined Route (UDR)."

Reply:
updated